src/less/page-notice/stories/page-notice.stories.js

```diff
@@ -151,8 +151,8 @@ export const dismissableWithTitle = () => `
 export const dismissableWithoutTitle = () => `
 <section class="page-notice page-notice--information" role="region" aria-label="Information">
     <div class="page-notice__header">
-        <svg class="icon icon--information-filled" focusable="false" height="24" width="24" role="img" aria-label="Information">
-            <use xlink:href="#icon-information-filled"></use>
+        <svg class="icon icon--information-filled-small" focusable="false" height="16" width="16" role="img" aria-label="Information">
+            <use xlink:href="#icon-information-filled-small"></use>
         </svg>
     </div>
     <div class="page-notice__main">
```

Comment:
Can you also add a `close-small` icon. I think the SVG already has a close small class but the actually use should be using `close-small`.

Reply:
I changed a couple of `...close` icons to `...close-small`. Is this what you meant?